tasklist/app/views/tasks/index.html.erb

```diff
@@ -4,14 +4,18 @@
 <html>
   <body>
     <h1> What's Da Happs </h1>
+    <h2><a class = "new" href="/tasks/new">add a new task</a></h2>
```

Comment:
In HTML, we don't want to add spaces on either side of the `=`, so instead this line would have `class="new"`